core/templates/dev/head/pages/review-test-page/review-test-engine.service.spec.ts

```diff
@@ -15,6 +15,7 @@
 /**
  * @fileoverview Unit tests for the review tests.
  */
+
```

Comment:
Remove?

Reply:
I think it should be there, wasn't there before.